tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -1337,13 +1338,27 @@ bool BaseCpuFusionStrategy::tryFuse(ShapeAnalysis& shapeAnalysis,
 ////////////////////// Base GPU FusionStrategy Implementation /////////
 //////////////////////////////////////////////////////////////////
 
+bool isSingleElementH2DOp(Operation* op) {
+  auto h2d_op = dyn_cast<lmhlo_disc::H2DOp>(op);
+  if (!h2d_op) return false;
+
+  int rank = op->getOperand(0).getType().cast<MemRefType>().getRank();
```

Comment:
using `getNumElements() == 1` to support other rank?

Reply:
done